sdk/search/azure-search/src/main/java/com/azure/search/models/SuggestResult.java

```diff
@@ -33,7 +34,7 @@
      *
      * @return the additionalProperties value.
      */
-    public Document getDocument() {
+    public Map<String, Object> getDocument() {
```

Comment:
Same question -- Shouldn't this be `SearchDocument` instead of just `Map`?

Reply:
Search result is auto-gen code. As @alzimmermsft brought up yesterday. Our auto-gen code is better not to depend on any manually craft APIs. That's why I have changed the return type here. Do we still want the Document here?

Reply:
Revert it back